backend/service/k8s/pods.go

```diff
@@ -27,13 +29,26 @@ func (s *svc) DescribePod(ctx context.Context, clientset, cluster, namespace, na
 	}
 
 	if len(pods.Items) == 1 {
-		return podDescription(&pods.Items[0], cs.Cluster()), nil
+		return podDescription(&pods.Items[0], cs.Cluster(), getPodEvents(&pods.Items[0], cs)), nil
```

Comment:
can we make the getPodEvents call concurrently? or is there pod information that's needed to then fetch events. wasn't clear to me is all.

Reply:
The search API call to events needs the pod object as an argument, so we need to retrieve the pod object first before we fetch the events.